Flask

# gunicorn
# Temporary work around to fix ALREADY_HANDLED issue bugging eventlet 0.30.3+
# Solution source: https://github.com/benoitc/gunicorn/pull/2581
https://github.com/benoitc/gunicorn/archive/ff58e0c6da83d5520916bc4cc109a529258d76e1.zip#egg=gunicorn==20.1.0

pymongo[srv]
Flask-Session
python-cas
redis
email-validator
password_strength
flask-apispec==0.11.0
flask-socketio
eventlet
random-username